src/dune_rules/module.ml

```diff
@@ -202,15 +202,24 @@ type t =
   ; visibility : Visibility.t
   ; kind : Kind.t
   ; install_as : Path.Local.t option
+  ; parameters : Module_name.t list
```

Comment:
Given that the list of parameters is constant for an entire stanza, keeping this information isn't the right place to do so. Could you try moving it to the compilation context instead?

Reply:
Thanks, I've added a commit to that effect. The only advantage of storing the parameters for each module was that we could skip instantiating the generated modules since they don't depend on the parameters (and maybe we could have shaved more modules in the future by using ocamldep), but I now think it's easier to have a uniform treatment of all the modules :)